docs/modules/user_guide/tree.rst

```diff
@@ -0,0 +1,116 @@
+.. _tree:
+
+===========================
+Unsupervised Decision Trees
+===========================
+
+.. currentmodule:: sklearn.tree
+
+In unsupervised learning, the goal is to identify patterns 
+or structure in data without using labeled examples. Clustering is a common 
+unsupervised learning technique that groups similar examples together 
+based on their features. In clustering settings, there may be no natural 
+notion of “true” class-labels, thus the efficacy of the clustering scheme is 
+often measured based on distance based metrics such as :func:`sklearn.metrics.adjusted_rand_score`.
+
+One way to combine clustering with decision trees is to use a clustering 
+algorithm to partition the data into groups, and then build a decision tree 
+on each group. This approach is known as clustering-based decision trees or 
+clustering trees. Each decision tree only needs to consider a subset of 
+the data, which can lead to faster and more accurate predictions. Two of 
+the most common apporaches used for unsupervised classification are 
+K-Means clustering and Mixture Modeling, which use the Expectation 
+Maximization Algorithm.
+
+In clustering settings, there may be no natural notion of “true”
+class-labels; the efficacy of the clustering scheme is often measured 
+in such cases by the economy in description length attained
+by a two-step description of the objects by first describing the 
+attributes common to the clusters and then describing the differential
+attributes of each object within the cluster. k-Means Clustering
+and Mixture Modeling using the Expectation Maximization Algorithm 
+are examples of techniques used for unsupervised classification.
```

Comment:
What does EM and KMeans have to do with clustering with trees? I'm not following most of this section.